dev.linwood.butterfly.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, you should maintain this file in your own repo and include it in the tarball.